ghcide/src/Development/IDE/Core/Shake.hs

```diff
@@ -257,8 +259,18 @@ data ShakeExtras = ShakeExtras
     ,diagnostics :: STMDiagnosticStore
     ,hiddenDiagnostics :: STMDiagnosticStore
     ,publishedDiagnostics :: STM.Map NormalizedUri [Diagnostic]
+
+    -- storing semantic tokens for each file in shakeExtras might
```

Comment:
this comment is in the wrong place, it's interrupting the haddock for the field above

Reply:
Maybe put it in a Note or something

Reply:
Sure. Did it